README.md

```diff
@@ -1,55 +1,118 @@
-# IntegriTEE Parachain:
+# Integritee Parachain
 
-This is the repository to run integritee as a parachain in the rococo-v1 testnet. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository and only adds the SubstraTEE pallet and configuration.
+This is the repository to run Integritee as a parachain on Kusama and Rococo. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository.
 
 ## Launch a local setup including a Relay Chain and a Parachain
 
 Quick: see [polkadot-launch](https://github.com/paritytech/polkadot-launch.git)
 ```
 node ../polkadot-launch/dist/cli.js launch-kusama-local-with-shell.json
 ```
+or
+```
+polkadot-launch launch-kusama-local-with-shell.json
+```
+depending on how you installed `polkadot-launch`.
 
+### Manually Launch a local Rococo Testnet
 
-### Manually Launch Local Rococo Testnet
+#### Launch a Rococo Relay Chain
 
+First, check out which polkadot release to use in the .json files in the [polkadot-launch](./polkadot-launch) folder:
+```json
+{
+	"relaychain": {
+		"bin": "../../../bin/polkadot-0.9.18", // <-- release to use
+    		"chain": "kusama-local",
+        // --snip--
+```
+Then get and build `polkadot`:
 ```bash
 # Compile Polkadot with the real overseer feature
 git clone https://github.com/paritytech/polkadot
-git checkout <release tag>  // release tag that matches the branch id in the polkadot-deps
+# Switch into the Polkadot directory
+cd polkadot
+# Checkout the proper commit
+git checkout <release tag>  # release tag, just like in the example above: polkadot-0.9.18
 cargo build --release
+# If this fails due to  `feature `edition2021` is required` you have set an outdated rust default version. To fix this, you can simply run:
+cargo +nightly-2022-01-31 build --release # or any other up-to-date nightly version
 
 # Generate a raw chain spec
 ./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode --raw > rococo-local-cfde.json
+```
+Start the `Alice` validator:
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --alice --validator --tmp --port 30333 --ws-port 9944
+```
+When the node starts you will see several log messages. Take note of the node's Peer ID in the logs. We will need it when connecting other nodes to it. It will look something like this:
+```bash
+Local node identity is: 12D3KooWGjsmVmZCM1jPtVNp6hRbbkGBK3LADYNniJAKJ19NUYiq
+```
+Start the `Bob` validator (in a second terminal):
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --validator --bob --bootnodes /ip4/<Alice IP>/tcp/30333/p2p/<Alice Peer ID> --tmp --port 30334 --ws-port 9945
+```
+
+More information can be found in the Substrate tutorial [Start a Relay Chain](https://docs.substrate.io/tutorials/v3/cumulus/start-relay/)
+
+#### Reserve a para ID
+Go to [Polkadot Apps](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/parachains/parathreads). Register the parachain via
```

Comment:
Here it is not clear which node we should connect to. The link points to 9944, which is the Alice validator. Is it necessary to connect to this node or to any other node in the relay chain? 
Some more information would be nice, in case we are using ports other than those indicated in the README.

Reply:
Any node of your relay chain is fine, as long as they are connected. I will add a comment for that.